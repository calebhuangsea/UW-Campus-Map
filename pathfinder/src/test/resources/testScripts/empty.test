# test find path in an empty graph

CreateGraph g

FindPath g A B