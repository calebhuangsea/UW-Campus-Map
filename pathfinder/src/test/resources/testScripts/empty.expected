# test find path in an empty graph

created graph g

unknown: A
unknown: B